components/eam/bld/namelist_files/namelist_defaults_eam.xml

```diff
@@ -849,6 +851,17 @@
 <micro_nicons         microphys="mg2">  0.1D6       </micro_nicons>
 <micro_mincdnc        microphys="mg2">  -999.       </micro_mincdnc>
 
+<!-- P3 specific namelist variables -->
+<micro_p3_lookup_dir	    > atm/cam/physprops </micro_p3_lookup_dir>
+<micro_p3_tableversion		      >  4.1.2 	</micro_p3_tableversion>
```

Comment:
I wonder if there is a better way to name this table version. It's hard to expect the version choices, for example, from 1.0.0 to x.y.z?

Reply:
This follows the table version from WRF. I would use the version name consistent with WRF for the same table so that we can track it easily if there is no other reason requiring a change of name.